components/browser/menu/src/main/res/layout/mozac_browser_menu_item_checkbox.xml

```diff
@@ -2,13 +2,33 @@
 <!-- This Source Code Form is subject to the terms of the Mozilla Public
    - License, v. 2.0. If a copy of the MPL was not distributed with this
    - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
-<androidx.appcompat.widget.AppCompatCheckBox xmlns:android="http://schemas.android.com/apk/res/android"
-    style="@style/Mozac.Browser.Menu.Item.Text"
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    style="@android:style/TextAppearance.Material.Menu"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:background="?selectableItemBackground"
     android:layout_width="match_parent"
-    android:layout_height="@dimen/mozac_browser_menu_item_container_layout_height"
-    android:button="@null"
-    android:drawableEnd="?android:attr/listChoiceIndicatorMultiple"
-    android:drawablePadding="12dp"
+    android:layout_height="48dp"
     android:gravity="center_vertical"
-    android:paddingStart="16dp"
-    android:paddingEnd="16dp" />
\ No newline at end of file
+    android:orientation="horizontal">
+
+    <TextView
+        android:id="@+id/text"
+        style="@style/Mozac.Browser.Menu.Item.Text"
+        android:background="@null"
+        android:clickable="false"
+        android:layout_width="wrap_content"
+        android:checkMark="@null"
+        android:layout_height="@dimen/mozac_browser_menu_item_container_layout_height"
+        android:gravity="start|center_vertical"
```

Comment:
Is gravity necessary since the parent has it set already?

Reply:
It seems necessary--when I remove it, it's no longer centered vertically. I can remove the "Start" though. 

<img width="755" alt="image" src="https://user-images.githubusercontent.com/4400286/60984711-079c1b80-a2f1-11e9-9653-06fd99779ed6.png">